<?xml version="1.0"?>
<launch>
   <node name="rbcar_steering_publish_node" pkg="rbcar_robot_control" type="rbcar_steering_publish_node" output="screen">
      <rosparam file="$(find rbcar_robot_control)/config/delay_params.yaml" command="load" />
   </node>
</launch>
